src/redis-cli.c

```diff
@@ -3076,6 +3079,7 @@ static void usage(int err) {
 "  --scan             List all keys using the SCAN command.\n"
 "  --pattern <pat>    Keys pattern when using the --scan, --bigkeys or --hotkeys\n"
 "                     options (default: *).\n"
+"  --count <count>    Count option when using the --scan, --bigkeys or --hotkeys (default: 10).\n"
```

Comment:
The default value is set to 500, but based on experience, the default value of 10 is too small.

Reply:
I agree that the default 10 is small. but the redis default count value for scan is 10, so I think the same value should be used.
I think other default value would be confusing.

Reply:
> redis default count value for scan is 10, so I think the same value should be used. I think other default value would be confusing.

Ok